gradle/plugins/src/main/kotlin/kotools/types/multiplatform.gradle.kts

```diff
@@ -26,6 +27,13 @@ rootProject.plugins.withType<YarnPlugin>().configureEach {
     rootProject.the<YarnRootExtension>().lockFileDirectory = projectDir
 }
 
+tasks.withType<KotlinCompile>().configureEach {
+    kotlinOptions {
+        freeCompilerArgs += "-opt-in=kotlin.RequiresOptIn"
+        languageVersion = "1.5"
```

Comment:
This version should be defined in the [`libs.versions.toml`](https://github.com/kotools/types/blob/ffb507e40120881cfeb6690e9e3c71aad66c22c1/gradle/libs.versions.toml#L1-L5) file instead with the `kotlin-language` key.

Reply:
Because the `libs.versions.toml` file is used for centralizing versions of plugins and dependencies, we should store this version in the [`gradle.properties`](https://github.com/kotools/types/blob/ffb507e40120881cfeb6690e9e3c71aad66c22c1/gradle.properties) file instead with the `kotlin.language.version` key.